### PR TITLE
Ch119/create model and import geolocation data

### PR DESCRIPTION
**Pull request by AlineRibeiro**

### CODE REVIEW

File: db/migrate/20210810192657_create_geolocations.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class CreateGeolocations < ActiveRecord::Migration[6.1]
+  def change
+    create_table :geolocations do |t|
+      t.integer :ip_from, limit: 8
+      t.integer :ip_to, limit: 8
```
Comment (guialbuk) [inline db/migrate/20210810192657_create_geolocations.rb:7]: 👏

File: db/seeds.rb [hunk 2]
```diff
@@ -18,3 +20,11 @@
 3.times do
   VideoStreamEvent.create!({ duration: 2.5, video_id: video.id, user_id: user.id, created_at: 10.minutes.ago })
 end
+
+csv_text = File.read(Rails.root.join('db/geolocation.csv'))
+csv = CSV.parse(csv_text, headers: true)
+locations = []
+csv.each do |row|
+  locations << Geolocation.new(row.to_hash)
```
Comment (guialbuk) [inline db/seeds.rb:28]: Loved the `row.to_hash` usage. It makes the code so concise 😍

File: spec/factories/geolocations.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :geolocation do
+    ip_from { 1 }
+    ip_to { 1 }
+    country_code { 'MyString' }
+    country { 'MyString' }
+    region { 'MyString' }
+    city { 'MyString' }
```
Comment (guialbuk) [inline spec/factories/geolocations.rb]: Can you please replace `MyString` with values more related to the actual data?

**Review by guialbuk — APPROVED** [review]
💯

File: spec/factories/geolocations.rb (hunk 3 above)
Comment (AlineRibeiro) [reply spec/factories/geolocations.rb]: Done! Thank you.

**Pull request merged**